server/sonar-ce/src/main/java/org/sonar/ce/container/ComputeEngineContainerImpl.java

```diff
@@ -420,7 +421,8 @@ protected ComponentContainer getComponentContainer() {
       LogServerId.class,
       ServerLifecycleNotifier.class,
       PurgeCeActivities.class,
-      CeQueueCleaner.class
+      CeQueueCleaner.class,
+      CeWorkerCountSettingWarning.class
```

Comment:
As the component is declared in the CE container, the warning will only be seen when executing a CE task.
I think this will be hard for users to see it.
Why not put in the web container in order to see the warning in web.log, at startup ?